openapi.yaml

```diff
@@ -25,254 +37,677 @@ security:
+  DoughRecipe:
     properties:
-      data:
-        type: object
-        properties:
-          type:
-            $ref: '#/definitions/PetType'
-          attributes:
-            type: object
-            properties:
-              name:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/name'
-              owner:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/owner'
-              species:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/species'
-            required:
-              - name
-              - owner
-              - species
-            additionalProperties: false
-        required:
-          - type
-          - attributes
-        additionalProperties: false
-    required:
-     - data
-    additionalProperties: false
-  PetType:
-    type: string
-    enum: [pet]
+      id:
+        type: string
+        description: a unique ID of a dough recipe
+      attributes:
+        $ref: "#/definitions/DoughAttributes"
+  DoughAttributes:
+    type: object
+    properties:
+      name:
+        type: string
+        description: the name of the dough recipe
+        example: weeknight pizza dough
+      gramsFlour:
+        type: integer
+        description: the amount of flour used, in grams
+        example: 500
+      flourType:
+        type: string
+        description: the name of the kind of flour used
+        example: All Purpose
+      gramsWater:
+        type: integer
+        description: the amount of water used, in grams
+        example: 400
+      waterTemp:
+        type: integer
+        description: the temperature in Fahrenheit to heat the water to
+        example: 90
+      gramsYeast:
+        type: integer
+        description: the amount of yeast to use, in grams
+        example: 5
+      gramsSalt:
+        type: integer
+        description: the amount of salt to use, in grams
+        example: 15
+      gramsSugar:
+        type: integer
+        description: the amount of sugar to use, in grams
+        example: 20
+      gramsOliveOil:
+        type: integer
+        description: the amount of olive oil to use, in grams
+        example: 50
+      bulkFermentTime:
+        type: integer
+        description: the time in minutes to spend on the bulk ferment
+        example: 60
+      proofTime:
+        type: integer
+        description: the time in minutes to spend on the proof
+        example: 15
+      specialInstructions:
+        type: string
+        description: any additional instructions not captured in the main object
+        example: keep the dough in the fridge during the bulk ferment
+  ErrorResult:
+    properties:
+      errors:
+        type: array
+        items:
+          $ref: "#/definitions/ErrorObject"
   ErrorObject:
     properties:
       status:
         type: string
         description: HTTP status code
-        example: '123'
+        example: "123"
```

Comment:
auto-corrected to double quotes by my linter, could probably change this is it's an issue